src/OVAL/probes/SEAP/generic/rbt/rbt_common.c

```diff
@@ -372,13 +379,17 @@ int rbt_walk_inorder(rbt_t *rbt, int (*callback)(void *), rbt_walk_t flags)
         register uint8_t depth, delta;
         register int     r;
 
-        assume_d(rbt != NULL, -1);
+	if (rbt == NULL) {
+		return -1;
+	}
 
         if (flags & RBT_WALK_RAWNODE)
                 delta = 0;
         else {
                 delta = sizeof(void *) * 2;
-                assume_d((sizeof(void *) * 2) == (size_t)(((struct rbt_node *)(NULL))->_node), -1);
+		if ((sizeof(void *) * 2) != (size_t)(((struct rbt_node *)(NULL))->_node)) {
```

Comment:
I have discussed this with @matusmarhefka who tested this in a code snippet outside of openscap and it really segfaults. I wonder how it could be here for 8 years. Maybe this code is never run or rarely run.

Reply:
I am going to remove them